tidb-server/main.go

```diff
@@ -48,7 +48,7 @@ import (
 	"github.com/pingcap/tidb/util/printer"
 	"github.com/pingcap/tidb/util/systimemon"
 	"github.com/pingcap/tidb/x-server"
-	binlog "github.com/pingcap/tipb/go-binlog"
+	"github.com/pingcap/tipb/go-binlog"
```

Comment:
This useless alias is used for some contributors to read the code. You could check its commit log.
Btw, I want to remove it either.

Reply:
.....I didn't modify it...it seem idea or make fmt change it :rofl:  I will rollback it.